homeassistant/components/shelly/utils.py

```diff
@@ -44,15 +44,23 @@ def async_remove_shelly_entity(
 
 
 def get_block_device_name(device: BlockDevice) -> str:
-    """Naming for device."""
+    """Get Block device name."""
```

Comment:
This will be removed in a follow up PR to use the device name from the coordinator (limited the PR to coordinator changes only)